bqetl_project.yaml

```diff
@@ -193,12 +193,9 @@ dry_run:
 format:
   skip:
   - bigquery_etl/glam/templates/*.sql
-  - sql_generators/events_daily/templates/**/*.sql
-  - sql_generators/glean_usage/templates/*.sql
-  - sql_generators/search/templates/*.sql
-  - sql_generators/experiment_monitoring/templates/**/*.sql
-  - sql_generators/feature_usage/templates/*.sql
-  - sql_generators/funnels/templates/*.sql
+  - sql_generators/**/*.sql
+  - tests/**/*.sql
+  - sql/mozfun/**/examples/*.sql
```

Comment:
I think there's still some value in formatting some of these.  Could we have a instead have a separate category like `skip_qualifying_references`that gets used in `format.py`?

Reply:
I added a config option for this and added the udf examples folders